README.md

```diff
@@ -52,33 +114,47 @@ task :ci => ['engine_cart:generate'] do
 end
 ```
 
-And in your e.g. spec\_helper.rb (or rails\_helper.rb), initialize EngineCart:
+And in your engine's test framework configuration (e.g. spec\_helper.rb or rails\_helper.rb), initialize EngineCart:
 
 ```ruby
   require 'engine_cart'
   EngineCart.load_application!
 ```
 
-When you want to clean out and regenerate the test application, you can call:
+Your test files (e.g. spec files for your engine) can now be written as tests for a Rails application.
+
+## Gotcha
+
+If you have generated a test application, there is a Gemfile and Gemfile.lock associated with the testing app at spec/internal (or wherever you designated as the test app location).  If you then update *your* engine's Gemfile or gemspec, Bundler can get confused if it has conflicting information between your engine and the testing app.
+
+To fix this:
+
+1. clean out your testing app `$ rm spec/internal`  # rake may not be avail.
```

Comment:
It is a real problem:

1. I have a testing app at spec/internal
2. I make a change to my engine's Gemfile (changing a gem's version restrictions perhaps)
3. I run `bundle install` ... but it won't work due to gem version conflict
4. I can't run `rake engine_cart:clean` because bundler's gem version conflict interferes with rake
5. I can't run `bundle exec bundle install`  or `bundle exec rake engine_cart:clean`  either.

Perhaps you always are sharp enough to run `engine_cart:clean` before you fiddle with your engine's Gemfile, but many of us aren't.  This problem has happened to me many times.

I can try to clarify "how to avoid this" in the Gotcha section, and I can also create the "if all else fails" heading for the brute force approach, if that would assuage your concerns, Chris.